mne/io/eeglab/eeglab.py

```diff
@@ -576,19 +585,74 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
         logger.info('Ready.')
 
 
-def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
-    """Create events array from EEGLAB structure.
+def read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer',
+                       uint16_codec=None):
+    r"""Create events array from EEGLAB structure.
```

Comment:
yes, but do we really need it? I noticed a backslash and I think that was meant to be escaped, no?